### PR TITLE
Enhancements for 0.9.13

### PR DESCRIPTION
**Pull request by greenaar**
Expose Prefetch cache (0.9.13)
Expose bulk query (0.9.13)

Expose carbonlink timeout (any)
Expose remote render hosts/timeout (any)

I tried to match your configuration style.

### CODE REVIEW

File: templates/opt/graphite/webapp/graphite/local_settings.py.erb [hunk 1]
```diff
@@ -231,6 +242,11 @@ REMOTE_FIND_CACHE_DURATION = <%= scope.lookupvar('graphite::gr_cluster_cache_dur
 # access to metric data or should have CLUSTER_SERVERS configured
 #RENDERING_HOSTS = []
 #REMOTE_RENDER_CONNECT_TIMEOUT = 1.0
+<% if [:undef, nil].include? scope.lookupvar('graphite::gr_rendering_hosts') -%>
```
Comment (dwerder) [inline templates/opt/graphite/webapp/graphite/local_settings.py.erb]: I assume you meant "unless" or "  ![:undef, nil].incl..." at this point ;-)

File: templates/opt/graphite/webapp/graphite/local_settings.py.erb [hunk 2]
```diff
@@ -243,6 +259,15 @@ REMOTE_FIND_CACHE_DURATION = <%= scope.lookupvar('graphite::gr_cluster_cache_dur
...
 <% else -%>
 CARBONLINK_HOSTS = ['<%= scope.lookupvar('graphite::gr_carbonlink_hosts').join("','") %>']
+CARBONLINK_TIMEOUT = <%= scope.lookupvar('graphite::gr_carbonlink_hosts_timeout') %>
+<% end -%>
+
+# Using 'query-bulk' queries for carbon
+# It's more effective, but python-carbon 0.9.13 (or latest from 0.9.x branch) is required
+# See https://github.com/graphite-project/carbon/pull/132 for details
+#CARBONLINK_QUERY_BULK = False
+<% if [:undef, nil].include? scope.lookupvar('graphite::gr_carbonlink_query_bulk') -%>
```
Comment (dwerder) [inline templates/opt/graphite/webapp/graphite/local_settings.py.erb]: unless instead of if

**Comment by dwerder** [comment]
Were all this option introduced in 0.9.13?

**Comment by dwerder** [comment]
The CI is red because of the "if / unless" mistake. Can you please fix that.

**Comment by greenaar** [comment]
Will correct, thanks.

**Comment by greenaar** [comment]
Bulk query and prefetch are new in 0.9.13, the other items (carbonlink timeout and remote render host) are present in earlier versions.

**Pull request merged**